Jenkinsfile

```
@@ -137,6 +137,12 @@ pipeline {
     }
     stage('e2e tests') {
       agent { label 'nixos-mayastor' }
+      environment {
+        GIT_COMMIT_SHORT = sh(
+          script: "printf \$(git rev-parse --short ${GIT_COMMIT})",
```

I was going to ask why the `printf` but that's to get rid of whitespace right? What about comment for dummies like me?
```suggestion
          // using printf to get rid of trailing newline
          script: "printf \$(git rev-parse --short ${GIT_COMMIT})",
```